lib/sen_tweet/metrics.ex

```diff
@@ -43,16 +43,20 @@ defmodule SenTweet.Metrics do
   defp handle_event({"bitfeels_pipeline_sentiment", id, score, time}, metrics) do
```

Comment:
We do want to handle nil scores but since this data is so critical to the function, let's use a guard in the function header.
```
defp handle_event({"bitfeels_pipeline_sentiment", id, score, time}, metrics) when is_float(score) do
  # just metrics goodness
end
```
Then we can add other function that works as a catch all for `handle_event/2`
```
defp handle_event(_data, metrics) do
  metrics
end
```

Reply:
this is an amazing feature